ci/run-dockerized.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u -x
+
+image_name=$1
+
+if [[ ${2-} == "--dry-run" || ${CIRCLE_BRANCH-} != "master" ]]; then
+  do_dput=false
```

Comment:
In the same spirit as in https://github.com/VirtusLab/git-machete/pull/182/files#r694837997 ... I'd avoid making "switches" against the individual known cases... I'd rather provide a set of reusable scripts/functions.

This can in particular mean that the specific `ci-run.sh` scripts in ci/{tox,rpm,...} folders might need to be restored... which is okay IMO.